### PR TITLE
chore(global): restructured repo & adopted sso sample

### PR DESCRIPTION
**Pull request by megglos**
relates to #OPT-4689

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,29 +10,22 @@
...
+    <module>plugins</module>
     <module>kafka-event-adapter</module>
-    <module>businesskey-import-plugin</module>
   </modules>
 
   <properties>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <optimize.version>3.3.0-SNAPSHOT</optimize.version>
+    <optimize.version>${project.version}</optimize.version>
```
Comment (megglos) [inline pom.xml:20]: needs to be set manually to 3.2.0 for testing the sso sample

File: plugins/variable-import-plugins/anonymize-variables/README.md [hunk 2]
```diff
@@ -167,7 +167,7 @@ When Optimize is now started, it should automatically anonymize variables with
 the name `customerName`. All other variables are still imported to Optimize (if they are not complex or 
 binary variables).
 
-[1]: ../../docs/optimize-variable-import.png
+[1]: ../docs/optimize-variable-import.png
```
Comment (RomanJRW) [inline plugins/variable-import-plugins/anonymize-variables/README.md]: is this even used?

File: optimize-sso-keycloak-example/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+1. Build all images with `docker-compose build`
+2. Login to private Camunda Docker EE Registry with `docker login registry.camunda.cloud` Use your EE LDAP credentials to log in.
+3. Start all images `docker-compose up -d`
+
+## Open Optimize
+
+1. Open WebBrowser
+2. Open `http://localhost:8095`
+3. Login with: `demo:notdemo`
```
Comment (RomanJRW) [inline optimize-sso-keycloak-example/README.md]: It be enough to simply say something like:

```suggestion
Open `http://localhost:8095` in a web browser and log in `demo:notdemo`
```

**Review by RomanJRW — APPROVED** [review]
LGTM ✅

File: plugins/variable-import-plugins/anonymize-variables/README.md (hunk 2 above)
Comment (megglos) [reply plugins/variable-import-plugins/anonymize-variables/README.md]: nice catch will remove it

**Pull request merged**